app/controllers/channel_transfer_controller.rb

```diff
@@ -14,7 +14,7 @@ def reject_transfer
 
   def verify_token
     @current_channel = Channel.find(params[:id])
-    if @current_channel.nil? || @current_channel.contest_token.empty? || @current_channel.contest_token != params[:token_id]
+    if @current_channel.nil? || @current_channel.contest_token.nil? || @current_channel.contest_token != params[:token_id]
```

Comment:
Let's use the `.blank?` method instead as it covers the most cases for when an object is `nil`, `""`, `" "` or some other edge case.
```suggestion
    if @current_channel.nil? || @current_channel.contest_token.blank? || @current_channel.contest_token != params[:token_id]
```
Here's a handy chart for the difference s between, `nil?`, `empty?`, and `blank?`
<img src="https://i.stack.imgur.com/oXuhJ.png" />